addons/event/tests/test_event_internals.py

```diff
@@ -34,6 +34,7 @@ def test_event_date_computation(self):
             'registration_ids': [(0, 0, {'partner_id': self.event_customer.id, 'name': 'test_reg'})],
             'date_begin': datetime(2020, 1, 31, 15, 0, 0),
             'date_end': datetime(2020, 4, 5, 18, 0, 0),
+            'note': '<p>Event Note</p>'
```

Comment:
Would not do it in a test method called ``test_event_date_computation`` ^^ either in "sync with template", either its own small method, as you prefer (maybe a small unit test and include some "false positive" void content like suggested before ?)

Reply:
Oh yeah did not notice the name, code has now been moved to a small method dedicated for this.